faktory_exporter.go

```diff
@@ -261,7 +261,11 @@ func (e *Exporter) scrape() (err error) {
 	e.jobs.WithLabelValues("processed").Set(processed)
 	e.totalQueues.Set(totalQueues)
 
-	queues := faktory["queues"].(map[string]interface{})
+	queues, ok := faktory["queues"].(map[string]interface{})
+	if !ok {
+		return fmt.Errorf("error getting queue counts")
+	}
```

Comment:
Given that this field won't be there for older faktory versions, I'd be a little happier putting these metrics in a conditional that checks for the presence of the field rather than failing whenever it's not present. While we still get all previous metrics even if we return an error, I'm a little worried about what the `e.up.Set(0)` and `e.failedScrapes.Inc()` in the defer will do if they happen on every request.

Reply:
The return here causes the calling function to log the returned error and carry on, so the periodic code will be fine.
I'm going to leave it as is and not deviate from how the rest of the metrics are handled.

Reply:
:+1: